packages/slider/src/Slider.ts

```diff
@@ -455,4 +523,18 @@ export class Slider extends SizedMixin(ObserveSlotText(SliderHandle, ''), {
         await this.handleController.handleUpdatesComplete();
         return complete;
     }
+
+    protected override updated(changes: PropertyValues<this>): void {
+        super.updated(changes);
+        if (changes.has('value') && changes.has('fillStart')) {
+            this._cachedValue = Number(this.value);
+            if (typeof this.fillStart === 'number') {
+                this.fillStartPoint = this.fillStart;
+            } else {
+                this.fillStartPoint =
+                    (Number(this.max) - Number(this.min)) / 2 +
+                    Number(this.min);
+            }
+        }
```

Comment:
Could this just be a getter instead? That would allow us to not place `fillStartPoint` as a reactive property, as well.

Reply:
Now using the `fillStartPoint` getter
```
private get fillStartPoint(): number {
        if (this.fillStart) {
            return Number(this.fillStart);
        } else {
            return (Number(this.max) - Number(this.min)) / 2 + Number(this.min);
        }
    }
```